services/blockchain-app-registry/tests/functional/shared/metadataIndex.test.js

```diff
@@ -0,0 +1,449 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+/* eslint-disable mocha/max-top-level-suites */
+jest.setTimeout(15000);
```

Comment:
Maybe we can remove this since we are already setting timeout of 15000 globally.
https://github.com/LiskHQ/lisk-service/blob/636e2c8a639f92e016d814c6394b76bc828c65f6/services/blockchain-app-registry/jest.config.functional.js#L11

Reply:
Applied.